agoradatatools/etl/load.py

```diff
@@ -51,22 +53,22 @@ def remove_non_values(d: dict) -> dict:
                 cleaned_dict[key] = nested_dict
         # case 2: list
         if isinstance(value, list):
-            for elem in value: # value is a list
+            for elem in value:  # value is a list
                 if isinstance(elem, dict):
                     nested_dict = remove_non_values(elem)
                 else:
                     cleaned_dict[key] = value
         # case 3: None/NaN
         elif pd.isna(value) or value is None:
             continue
-        #case 4: regular element
+        # case 4: regular element
         elif value is not None:
             cleaned_dict[key] = value
 
     return cleaned_dict
 
 
-def load(file_path: str, provenance: list[str], destination: str, syn=None):
```

Comment:
removed the `[str]` to allow testing to pass with python 3.7 and 3.8.

Reply:
Thats fine. For Python 3.7 and 3.8, you do this:

```
from typing import List

def load(...., provenance: List[str]...)
```